specification/billing/resource-manager/Microsoft.Billing/preview/2018-03-01-preview/examples/InvoicesExpand.json

```diff
@@ -0,0 +1,34 @@
+{
+  "parameters": {
+    "api-version": "2018-03-01-preview",
+    "subscriptionId": "subid",
+    "$top": "1",
```

Comment:
This type is mismatched, per spec it should be an int not a string, and is causing model validation [failure](https://travis-ci.org/Azure/azure-rest-api-specs/jobs/344048908).

Reply:
Fixed